src/schema/schema_create.c

```diff
@@ -961,6 +977,103 @@ __create_data_source(
     return (dsrc->create(dsrc, &session->iface, uri, (WT_CONFIG_ARG *)cfg));
 }
 
+/*
+ * __import_entry_cmp --
+ *     Qsort function: sort the import entries array by name.
+ */
+static int WT_CDECL
+__import_entry_cmp(const void *a, const void *b)
+{
+    WT_IMPORT_ENTRY *ae, *be;
+
+    ae = (WT_IMPORT_ENTRY *)a;
+    be = (WT_IMPORT_ENTRY *)b;
+
+    return strcmp(ae->name, be->name);
+}
+
+/*
+ * __get_uri_suffix --
+ *     Get suffix of the URI.
+ */
+static int
+__get_uri_suffix(WT_SESSION_IMPL *session, const char *uri, const char **suffix)
+{
+    *suffix = strchr(uri, ':');
+    if (*suffix == NULL || (++(*suffix))[0] == '\0')
+        WT_RET_MSG(session, WT_ERROR, "%s invalid URI", uri);
+
+    return (0);
+}
```

Comment:
I think this function is not necessary. There are a lot of places in the code where we do something very similar. Here's code from `schema_rename.c:120` and I think the calling code should be able to make the same assumptions:
```
    suffix = strchr(name, ':');
    /* An existing table should have a well formed name. */
    WT_ASSERT(session, suffix != NULL);
```

Reply:
Removed `__get_uri_suffix`: a2c19361c654cebe12390481b2f49007d3d8b74d. WT code is not consistent. In most of the places we just use asserts but in several occasions we fail: https://github.com/wiredtiger/wiredtiger/blob/94d6deff59d780671bb1f59cfe969913eb550dd2/src/schema/schema_create.c#L572